pack/fhv/fhvc.json

```diff
@@ -104,9 +104,359 @@
+    "code": "10550",
+    "doom": 8,
+    "double_sided": true,
+    "encounter_code": "the_silent_heath",
+    "encounter_position": 2,
+    "faction_code": "mythos",
+    "flavor": "The dry landscape is perfectly still. A beguiling glow hangs over everything.",
+    "illustrator": "Carlos Palma Cruchaga",
+    "name": "Unsettling Silence",
+    "pack_code": "fhvc",
+    "position": 50,
+    "quantity": 1,
+    "stage": 1,
+    "text": "<b>Forced</b> - When doom is placed on this agenda, if the Brood Queen enemy is set aside: Reveal tokens from the chaos bag equal to the number of [[Insect]] enemies in the victory display. If a [skull] or [auto_fail] token is revealed, spawn the set-aside Brood Queen enemy.",
+    "type_code": "agenda"
+  },
+  {
+    "back_flavor": "A swarm of insect soldiers descends upon you, blocking any escape. One of them buries its proboscis in your torso with near-mechanical efficiency. You feel pain, then numbness, then nothing as you pass out.\nYou come to hours later, covered in salt, in a dry cavern. Head pounding, you stagger out of the warrens into the arid heath. The wound in your side oozes a filmy blue poison, but it doesn't seem to be fatal...",
+    "back_name": "Swarmed!",
+    "back_text": "Each investigator who has not been eliminated is defeated and suffers 1 physical trauma.",
+    "code": "10551",
+    "doom": 7,
+    "double_sided": true,
+    "encounter_code": "the_silent_heath",
+    "encounter_position": 3,
+    "faction_code": "mythos",
+    "flavor": "The insects scurry to and fro, alarmed by your presence.",
+    "illustrator": "Carlos Palma Cruchaga",
+    "name": "Desolation (v.I)",
+    "pack_code": "fhvc",
+    "position": 51,
+    "quantity": 1,
+    "stage": 2,
+    "text": "Each [[Insect]] enemy loses aloof.\n<b>Forced</b> - When doom is placed on this agenda, if the Brood Queen enemy is set aside: Reveal tokens from the chaos bag equal to the number of [[Insect]] enemies in the victory display. If a token with a symbol is revealed, spawn the set-aside Brood Queen enemy.",
+    "type_code": "agenda"
+  },
+  {
+    "back_flavor": "A swarm of insects descends upon you, blocking any escape. One of them darts forward with almost mechanical efficiency, burying a proboscis in your torso. Sharp pain spasms through your body, followed by numbness. The milky white insect retracts a black proboscis dripping blue fluid as your vision begins to swim. Passing out is a mercy.\nYou come to hours later, covered in salt, in a dry chamber underground. Head pounding, you stagger out of the warrens into the arid heath. The wound in your side oozes a filmy blue poison, but it doesn't seem to be fatal...",
+    "back_name": "Swarmed!",
+    "back_text": "Each investigator who has not been eliminated is defeated and suffers 1 physical trauma.",
```

Comment:
```suggestion
    "back_text": "Each surviving investigator who has not been eliminated is defeated and suffers 1 physical trauma.",
```

Reply:
This is such a weird one, because 'who has not been eliminated' covers 'surviving', but I guess for accuracy we have to include it...